db/structure.sql

```diff
@@ -42,6 +29,17 @@ CREATE TYPE public.boost_options AS ENUM (
 );
 
 
+--
+-- Name: masto_cw_options; Type: TYPE; Schema: public; Owner: -
+--
+
+CREATE TYPE public.masto_cw_options AS ENUM (
```

Comment:
If you only keep this block and the last two changes to this file (the one to the users table and the one to the schema migrations), I imagine this will work on 9.6 in travis :)

(In other words, remove the default_with_oids change and three changes before this one)

Reply:
Done! Incidentally, would you like me to submit a pull request to switch from Travis CI to GitHub Actions? [Here's a blog post explaining why that's a good idea.](https://blog.earthly.dev/migrating-from-travis/)

Reply:
Oh, if you have the time I'd love that, I looked quickly into it a few weeks ago but never did the move :) Thanks!